doc: adding a warning regarding the renaming of some plugins (#286)
gravitee-io/release: upgrades/3.x/3.10.0/README.adoc

= Upgrade to 3.10.0

== Breaking changes

> Starting with this version, the naming of some components of Gravitee.io APIM change.

Reviewer: ```suggestion
From this version, the naming of some components of Gravitee.io APIM changes.
```

As a consequence, the following plugins are renamed :

[cols="1,1"]
|===
|before 3.10.0|after 3.10.0

|gravitee.repository.mongodb-x.y.z.zip
|gravitee.*apim*.repository.mongodb-x.y.z.zip

|gravitee.repository.jdbc-x.y.z.zip
|gravitee.*apim*.repository.jdbc-x.y.z.zip

|gravitee.repository.redis-x.y.z.zip
|gravitee.*apim*.repository.redis-x.y.z.zip

|gravitee.repository.hazelcast-x.y.z.zip
|gravitee.*apim*.repository.hazelcast-x.y.z.zip

|gravitee.repository.gateway.bridge.http.client-x.y.z.zip
|gravitee.*apim*.repository.gateway.bridge.http.client-x.y.z.zip

|gravitee.repository.gateway.bridge.http.server-x.y.z.zip
|gravitee.*apim*.repository.gateway.bridge.http.server-x.y.z.zip
|===

These plugins have also been moved in another folder on https://download.gravitee.io.
For instance, the MongoBD plugin is now available using this link:

https://download.gravitee.io/graviteeio-apim/plugins/repositories/gravitee-apim-repository-mongodb/gravitee-apim-repository-mongodb-3.10.0.zip

> WARNING: In future versions, others plugins will be renamed. Stay tune!

Reviewer: ```suggestion
WARNING: In future versions, others plugins will be renamed. Stay tuned!
```
